extension-iap/src/iap_android.cpp

```diff
@@ -499,42 +469,35 @@ static dmExtension::Result InitializeIAP(dmExtension::Params* params)
     return dmExtension::RESULT_OK;
 }
 
-static dmExtension::Result UpdateIAP(dmExtension::Params* params)
+static void IAP_OnCommand(IAPCommand* cmd, void*)
```

Comment:
In theory, if all the data is setup properly from when pushing the command, these handlers might be implemented in one place (as opposed to once for each platform)